src/main/java/org/sefgobal/dataholder/api/YoutubeAPI.java

```diff
@@ -17,8 +17,9 @@
 public class YoutubeAPI {
 
     private static final String MAX_RESULTS = "10";
```

Comment:
Since the youtube API has a default max result (5) and we can provide a custom value for the maxResult I think we can safely remove this now. What do you think?
If yes, we have to remove it from the default URL as well. ( line 27 and 28 )